editoast/src/views/infra/objects.rs

```diff
@@ -37,17 +38,16 @@ fn has_unique_ids(obj_ids: &Vec<String>) -> bool {
     obj_ids_2.len() == obj_ids.len()
 }
 
-#[derive(QueryableByName, Debug, Clone, Serialize, Deserialize)]
+#[derive(QueryableByName, Debug, Clone, Serialize, Deserialize, PartialEq)]
 struct ObjectQueryable {
     #[diesel(sql_type = Text)]
-    #[serde(skip_serializing)]
     obj_id: String,
     #[diesel(sql_type = Jsonb)]
     railjson: JsonValue,
     #[diesel(sql_type = Nullable<Jsonb>)]
-    geographic: Option<JsonValue>,
+    geographic: Option<DieselJson<GeoJson>>,
     #[diesel(sql_type = Nullable<Jsonb>)]
-    schematic: Option<JsonValue>,
+    schematic: Option<DieselJson<GeoJson>>,
```

Comment:
We have a crate `postgis_diesel` to represent GeoJSON more precisely. `GeoJson` is currently only used for mvt tiles generation, but we've added an extern crate to represent GeoJSON values and we should remove this type. But that's for another PR